src/common/utils/MetaKeyUtils.cpp

```diff
@@ -1324,4 +1337,138 @@ meta::cpp2::PartitionList MetaKeyUtils::parseDiskPartsVal(const folly::StringPie
   return partList;
 }
 
+const std::string& MetaKeyUtils::jobPrefix() {
+  return kJobTable;
+}
+
+std::string MetaKeyUtils::jobPrefix(GraphSpaceID spaceId) {
+  std::string key;
+  key.reserve(kJobTable.size() + sizeof(GraphSpaceID));
+  key.append(kJobTable.data(), kJobTable.size())
+      .append(reinterpret_cast<const char*>(&spaceId), sizeof(GraphSpaceID));
+  return key;
+}
+
+std::string MetaKeyUtils::jobKey(GraphSpaceID spaceID, JobID jobId) {
+  std::string key;
+  key.reserve(32);
```

Comment:
why 32?

Reply:
Haha, I moved this function from another file, I fixed it